metron-interface/metron-config/src/app/service/hdfs.service.ts

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 import { Injectable, Inject } from '@angular/core';
-import { HttpClient, HttpParams } from '@angular/common/http';
+import {HttpClient, HttpHeaders, HttpParams} from '@angular/common/http';
```

Comment:
Thank you very much for the quick fix and sorry for the inconveniences.

I see a small possible tweak here but it's not critical at all.

You're importing the `HttpHeaders` object but you're not using it in the code.

Reply:
Done.